### ISSUE TITLE
Update available notice on Beta

### ISSUE THREAD

**Issue by sebastienserre** [lando/lando#3666]
Hello,

When doing a rebuild, Lando is notifying about an available update... I think the notice should be shown only on stable version not beta/development one:
![2024-02-19_10-50](https://github.com/lando/lando/assets/3902621/6c9208e0-f8af-4ffa-99f4-23741d7a40fe)

The power user who are ready to test know where to find the really latest version.

**Comment by reynoldsalec** [comment]
@sebastienserre agreed it's not ideal; this update is a [fairly big change](https://lando.dev/blog/2024/01/16/v321-extended.html) to the Lando experience (particularly around plugin architecture and management), hence why we had an extended alpha pre-release period and are releasing a beta tag as a suggested release for all users, while trying to give them further warning that things may not be ROCK solid yet.

Now that we have some broader testing from users, we'll hopefully be able to make some fixes and get to a stable release by end of week.

Going forward, the `lando update` command will largely replace the need to do monolithic releases. Much as is the official policy today, running `lando update` will move all your plugins (including Lando Core) to the latest stable release, unless you've elected to receive edge releases.

So tl;dr, you're totally right this isn't desirable, but going forward we should be back to the regular pattern, with the improvement that you'll be able to run `lando update` anytime to receive updates to your specified stable/edge preference.

**Comment by reynoldsalec** [comment]
@sebastienserre definitely continue the convo here if you have more questions, I'm going to close this out.

Again, a big mea culpa from me, I definitely hesitated to release a beta as a suggested release for this reason, but in the end it seemed like the best "middle ground." Good news is that a lot of the feedback I'm getting so far seems pretty minor, so hopefully we'll be out of the woods here in a few days.
